python/cudf/cudf/core/window/rolling.py

```diff
@@ -488,7 +488,7 @@ def _window_to_window_sizes(self, window):
             return window
         else:
             return cudautils.window_sizes_from_offset(
-                self.obj.index._values.data_array_view, window
+                self.obj.index._values.data_array_view(mode="write"), window
```

Comment:
Need to acquire a spill lock here (for the duration of the `window_sizes_from_offset` call)

Reply:
Done 👍